mgrs.js

```diff
@@ -86,6 +86,9 @@ export function inverse(mgrs) {
 }
 
 export function toPoint(mgrs) {
+  if (mgrs === '') {
+    throw (`toPoint received a blank string`);
```

Comment:
this should be an error object, so either `throw new Error('toPoint received a blank string');` or maybe `throw new TypeError('toPoint received a blank string');`  you should never be using throw with a string in a public api

Reply:
Agreed!